jupyter_server/services/events/handlers.py

```diff
@@ -27,7 +28,7 @@ class SubscribeWebsocket(
 
     auth_resource = AUTH_RESOURCE
 
-    def pre_get(self):
+    async def pre_get(self):
```

Comment:
This is technically an API change, but a minor one. I could wrap this method in `ensure_async` in case folks have overridden this method already.